sigridci/sigridci-feedback-template.html

```diff
@@ -280,7 +280,7 @@ <h3>Baseline (@@@BASELINE_DATE)</h3>
                             <span class="separator">@@@MAINTAINABILITY_STARS_OVERALL</span>
                         </div>
                         <div>
-                            <h3>New code quality</h3>
+                            <h3>New/changed code</h3>
```

Comment:
Maybe we can say here: `New/changed code quality` to be consistent with how it was described before?

Reply:
You're right, let's make terminology consistent.